src/main/java/org/jitsi/videobridge/rest/PublicRESTBundleActivator.java

```diff
@@ -64,6 +64,8 @@
     public static final String JETTY_CORS_ALLOWED_ORIGINS
         = ".jetty.cors.allowedOrigins";
 
+    public static final String JETTY_TLS_PORT_PNAME = ".jetty.tls.port";
```

Comment:
Why are we not re-using the existing constant for this (same name in super)?

Reply:
Ups, I missed it. Fixing it.